### PR TITLE
Add shamilton to maintainers

### PR DESCRIPTION
**Pull request by SCOTT-HAMILTON**
###### Motivation for this change
Wasn't yet on the maintainers list.

###### Things done
Added Scott Hamilton aka me to the list.

### CODE REVIEW

**Comment by wamserma** [comment]
Please rename the PR or drop https://github.com/NixOS/nixpkgs/pull/87935/commits/9c7dcca4f4f272ad5c02389b8d86a1a06ea448bd from this PR.

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -6974,6 +6974,12 @@
     githubId = 1588288;
     name = "Shahrukh Khan";
   };
+  shamilton = {
+    name = "Scott Hamilton";
+    email = "sgn.hamilton@protonmail.com";
+    github = "SCOTT-HAMILTON";
+    githubId = 24496705;
+  };
```
Comment (wamserma) [inline maintainers/maintainer-list.nix:6982]: Please sort entries for better readability.

**Review by wamserma — CHANGES_REQUESTED** [review]

**Review by bbjubjub2494 — CHANGES_REQUESTED** [review]
your package should be inserted in the wider nixpkgs with a `callPackage` expression in `pkgs/top-level/all-packages.nix`

Also please confirm that the point of this PR is to init otpgen.

EDIT: #87887

**Comment by SCOTT-HAMILTON** [comment]
Moved to https://github.com/NixOS/nixpkgs/pull/87938

**Pull request closed without merging**